src/Sarif/VersionOne/Autogenerated/IRuleVersionOneVersionOne.cs

```diff
@@ -42,7 +42,7 @@ public partial interface IRuleVersionOneVersionOne
         /// <summary>
         /// A value specifying whether a rule is enabled.
         /// </summary>
-        RuleConfigurationVersionOne Configuration { get; }
```

Comment:
>RuleConfigurationVersionOne [](start = 8, length = 27)

No need to change any of these. There was no "reporting configuration" in version 1.